assets/js/blocks/product-results-count/style.scss

```diff
@@ -5,5 +5,21 @@
 	.woocommerce-result-count {
 		float: none;
 		font-size: inherit;
+		// reset for margin
+		margin: auto;
 	}
 }
+
+.woocommerce .wc-block-catalog-sorting {
+	.woocommerce-ordering {
+		margin: auto;
+	}
+}
+
+.woocommerce.woocommerce-shop .wc-block-breadcrumbs {
+	.woocommerce-breadcrumb {
+		margin: auto;
+		display: block;
+	}
+}
```

Comment:
Similar to the above, it could be moved to Breadcrumbs style file:
https://github.com/woocommerce/woocommerce-blocks/blob/trunk/assets/js/blocks/breadcrumbs/style.scss